b2/console_tool.py

```diff
@@ -344,6 +344,25 @@ def run(self, args):
         return 0
 
 
+class GetFileInfoAndDeleteFileVersion(Command):
```

Comment:
`delete_file_version_by_id` would be better than `get_file_info_and_delete_file_version`, but I feel that extending `delete_file_version` would be better from user point of view (though it makes the implementation slightly less clean)